tests/queries/0_stateless/01292_create_user.sql

```diff
@@ -52,6 +52,17 @@ SHOW CREATE USER u2_01292;
 SHOW CREATE USER u3_01292;
 SHOW CREATE USER u4_01292;
 SHOW CREATE USER u5_01292;
+
+SELECT '-- no passwords or hashes in query_log';
+SELECT query
+FROM system.query_log
+WHERE
+    query NOT LIKE '%query_log%' AND
```

Comment:
Missing filters. Ideally `event_date >= yesterday() AND current_database = currentDatabase()`